src/dynamo0p3.py

```diff
@@ -2683,7 +2630,11 @@ def generate(self):
             # kernel (enforce_bc_kernel) arguments
             if self._kern.name.lower() == "enforce_bc_code" and \
                unique_fs.orig_name.lower() == "any_space_1":
-                self.bc_kernel(unique_fs)
+                self.field_bcs_kernel(unique_fs)
+        # Add boundary dofs array to the operator boundary condition
+        # kernel (enforce_operator_bc_kernel) arguments
+        if self._kern.name.lower() == "enforce_operator_bc_code":
+            self.operator_bcs_kernel(fs_to)
```

Comment:
Good plan. I've done this in fce5d51. There is one sanity check just to make sure we have an operator to get the FS from. All others are within the `operator_bcs_kernel` routine. I've extended the tests to make sure we cover all of these checks.